spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1309,6 +1326,24 @@ def handle_carry(self, request: Trigger.Request, response: Trigger.Response) ->
         response.success, response.message = self.spot_wrapper.spot_arm.arm_carry()
         return response
 
+    def handle_open_gripper(self, request: Trigger.Request, response: Trigger.Response) -> Trigger.Response:
+        """ROS service handler to open the gripper."""
+        if self.spot_wrapper is None:
+            response.success = False
+            response.message = "Spot wrapper is undefined"
+            return response
+        response.success, response.message = self.spot_wrapper.spot_arm.gripper_open()
+        return response
+
+    def handle_close_gripper(self, request: Trigger.Request, response: Trigger.Response) -> Trigger.Response:
+        """ROS service handler to close the gripper."""
+        if self.spot_wrapper is None:
+            response.success = False
+            response.message = "Spot wrapper is undefined"
+            return response
+        response.success, response.message = self.spot_wrapper.spot_arm.gripper_close()
+        return response
```

Comment:
Should this be a decorator?

Reply:
ya i was having the same thought as i copy and pasted more trigger services into this file - ill make a ticket